helm/kubernetes-external-dns-chart/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: v0.5.2-3-g6c05028
 description: A Helm chart for the external-dns
 name: kubernetes-external-dns-chart
-version: 0.1.1
+version: 0.1.2-[[ .SHA ]]
```

Comment:
From the version we determine which channel to publish to. 
E.g. `0-1-stable` or `0-2-stable`. We maintained different channels in order to be able to deploy and release different versions to different providers (for now).

We need the SHA now because we could theoretically overwrite a version that is referenced in an existing/deployed channel on quay.io, without even merging to master.

FYI: The channel design will proabably change with `app-operator` though.

Reply:
I may ask dumb questions but why not `0.1-[[ .SHA ]]` then?

Reply:
Not a dumb question at all. Semantically `0.1.1-[[ .SHA ]]` doesn't hold more value than `0.1-[[ .SHA ]]` now. 

But we build the publishing pipeline in architect around SemVer format, so in order to not have changes there and land the fixes, we left it like it is.

Reply:
OK. This is confusing, but as I hear here and there this should go away soonish. Let's roll.